Netw4ppl/app/src/main/java/netw4ppl/ines/utils/PersonListAdapter.java

```diff
@@ -171,10 +169,33 @@ public View getView(int position, @Nullable View convertView, @NonNull ViewGroup
 
         this.lastPosition = position;
 
-        holder.mUniqueID.setText(person.getInfoByKey("unique_id"));
-        holder.mSex.setText(person.getInfoByKey("gender"));
-        holder.mFullName.setText(person.getInfoByKey("full_name"));
-        holder.mAge.setText(person.getInfoByKey("age"));
+        Iterator<String> keys = person.keys();
+        while(keys.hasNext()) {
```

Comment:
check if possible to :
- format the form in a parent class ie 
      get the list of Best Descriptive value id
      get the list of descriptive value ids
      get the hidden list keys
      
   Just call by ids in there